### DISCUSSION TITLE
Possible to have TX only on attiny 841 chip

### DISCUSSION THREAD

**Discussion by ahshah** [SpenceKonde/ATTinyCore#832] [Support / Q & A]
Hi folks, coming from the attiny85, I got used to having a TX-only serial interface. Is a similar thing possible with the 841? Would be great to have the pin as GPIO, or at minimum as an output. I tried using software serial, but that doesn't seem to be supported on the 841 line..

**Comment by SpenceKonde** [comment]
Yes implemented but not tested (841 serial may be busted for other reasons) in 2.0.-0

**Comment by ahshah** [comment]
How would I go about testing this?  I don't see the same options in the UI, under the tools menu. On the '85 you go under tools > Built in software serial. But theres no such option for the 841.
